src/axolotl/monkeypatch/relora.py

```diff
@@ -267,7 +267,7 @@ def get_lr(self) -> float:
         original = self.inner_schedule.get_lr()
         step = self.last_epoch
 
-        if step < self.relora_steps:
+        if step < self.relora_steps - self.warmup_steps:
```

Comment:
If `warmup_steps` is greater than `relora_steps`, would this never trigger?

Reply:
yeah, then it would only anneal, but I don't think there is a good solution in that case, that's just bad hyperparams.